code/index.html

```diff
@@ -4,34 +4,50 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Arvo:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <title>Cluedo</title>
   </head>
   <body>
+    <header class="header">
+      <h1>Cl<span class="red-text">u</span>edo</h1>
+      <h2>The <span class="gold-text">classic</span> mystery game</h2>
+    </header>
     <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+      <article class="killer-deck" id="killerCard" onclick="pickKiller()" >
+        <p class="heading">KILLER</p>
+        <img id="killerImage" class="killer-image" src="" />
         <p id="killerName"></p>
         <p id="killerAge"></p>
         <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
         <p id="killerDescription"></p>
+        <p id="favouriteWeapon"></p>
+        <div id="killerLoader" class="loader"></div>
       </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
+      <article class="weapon-deck" id="weaponCard" onclick="pickWeapon()">
+        <p class="heading">WEAPON</p>
+        <div class="weapon-img-div">
+          <img id="weaponImage" class="weapon-image" src="" />
+        </div>
         <p id="weaponName"></p>
         <p id="weaponWeight"></p>
+        <div id="weaponLoader" class="loader"></div>
       </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
+      <article class="room-deck" id="roomCard" onclick="pickRoom()">
+        <p class="heading">ROOM</p>
+        <div class="room-img-div">
+          <img id="roomImage" class="room-image" src="" />
+        </div>
         <p id="roomName"></p>
+        <div id="roomLoader" class="loader"></div>
```

Comment:
I'm just curious, when is this one used?

Reply:
This div is my card loader animation